### ISSUE TITLE
Community feedback handling

### ISSUE THREAD

**Issue by lorenzwalthert** [aws/sagemaker-python-sdk#4380]
Dozens of bug reports and feature requests (e.g. #4114) got closed recently with this text:

> Thanks for using SageMaker and taking the time to suggest ways to improve SageMaker Python SDK. We have added your feature request it to our backlog of feature requests and may consider putting it into future SDK versions. I will go ahead and close the issue now, please let me know if you have any more feedback. Let me know if you have any other questions.
> 

While I appreciate it that you put this issue in the internal backlog, closing it “as completed” in GitHub terminology seems not a good description or matching open-source conventions here at GitHub. A transparent backlog or roadmap for Sagemaker would be great. Other AWS services such as [container services](https://github.com/aws/containers-roadmap) habe public roadmaps, which are highly appreciated by the AWS developer community, as the number of stars in the repo show.

It also seems PRs from non-AWS employees are not reviewed, even [trivial ones](https://github.com/aws/sagemaker-python-sdk/pull/3125). While I can understand that you may want AWS employees to make contributions only, it would be nice if people who put in time to contribute their time would be explicitly handled or the README or CONTRIBUTING would signal that non-AWS employees contributions are not welcome.

A few issues I opened got fixed but not referenced in the PR or closed with the PR, so I found out by accident that they have been resolved, e.g. a [docker dependency issue](https://github.com/aws/sagemaker-python-sdk/issues/3613).

To me, that's not what a successful open-source engagement of a leading ML toolsuite should look like. Competitors of AWS such as Databricks proactively manage their issues, let their users take part in feature prioritisation and bug fixes, have transparent roadmaps and integrated external contributors etc. My org has significant spending on AWS usage and also for business support, to which I regularly submit links of GitHub issues I face, 90% were not resolved (after re-classifying from bug report to feature, IMO at least some times inappropriate) and are now transferred into some intransparent internal backlog. So that is not a very pleasant experience. I wonder if other users' experience is similar to mine.

**Comment by Ce11an** [comment]
I completely agree. Greater transparency is necessary. I sense that this repository functions more as "source available" rather than truly open source.

**Comment by lorenzwalthert** [comment]
cc @ShwetaSingh801.

**Comment by lorenzwalthert** [comment]
I think I found place better suited to request SageMaker features that are wider in scope than just the Python SDK: https://github.com/aws/amazon-sagemaker-feedback. Maybe this repo could point to it?

**Comment by mohamedzeidan2021** [comment]
**This issue is being closed as part of sunsetting SageMaker Python SDK 2.x.**

SDK 2.x is now in maintenance mode, and we are no longer actively triaging or fixing 2.x bugs. All new development is on SDK 3.x, a modular redesign with unified interfaces, separate namespaces, and standardized core functionality.

We'd encourage you to move to 3.x, where this area has been substantially reworked and many long-standing 2.x issues no longer apply. To get started:
- Install: `pip install sagemaker` (3.x is the default)
- Migration guide: https://github.com/aws/sagemaker-python-sdk/blob/master/migration.md
- Docs: https://sagemaker.readthedocs.io/

If you try 3.x and still hit this problem there, change this issue's version tag from 2.x to 3.x and reopen it, and we'll take a look.

If you need further help with 2.x specifically, please reach out to AWS Support: https://console.aws.amazon.com/support/home

**Comment by lorenzwalthert** [comment]
Another example of poor community handling... The issue has nothing to do with version 2 or 3 of SageMaker. I can't even re-open it. Keep upvoting 😀

**Comment by des1-gner** [comment]
@mohamedzeidan2021

**Comment by lorenzwalthert** [comment]
Go upvote @des1-gner 's (an amazing AWS employee) new issue at #6237.
